.ghci.semantic

```diff
@@ -0,0 +1,70 @@
+-- GHCI settings for semantic, collected by running cabal repl -v and checking out the flags cabal passes to ghc.
+-- These live here instead of script/repl for ease of commenting.
+-- These live here instead of .ghci so cabal repl remains unaffected.
+
+-- Basic verbosity
+:set -v1
+
+-- No optimizations
+:set -O0
+
+-- Compile to object code
+:set -fwrite-interface -fobject-code
```

Comment:
I’m being a bit opinionated in this file:

1. I think we want to be able to quit the repl and relaunch it quickly.
2. I think we don’t want it to load all of the code automatically, since we’re working in various projects.
3. I think we especially don’t want it to import all of the code automatically.

I think those are fair statements for working on a variety of tasks, but if your particular workflow is made arduous by this, it’s worth revisiting.